### ISSUE TITLE
Installing pytorch-lightning on Jetson Nano(ARM)

### ISSUE THREAD

**Issue by enod** [PyTorchLightning/pytorch-lightning#695]
#### What is your question?
Does pytorch-lightning support ARM processor installation? So far I have failed with pip installation.

#### What's your environment?

 - OS: Linux
 - Hardware: Jetson Nano
 - Packaging: Pip
 - Processor: ARM

**Comment by Borda** [comment]
I would say that lighting does not have any restrictions n architecture, so it depends on used dependences... could you please try to install from master and pass here error log since not everyone has Jetson Nano(ARM) at his disposal... :]

**Comment by enod** [comment]
I have managed to install and was able to run MNIST example. 

Install `pytorch` and `torchvision` via https://devtalk.nvidia.com/default/topic/1049071/pytorch-for-jetson-nano-version-1-3-0-now-available/ which provides prebuilt binary pip wheels. 

And install scipy with `sudo apt-get install python3-scipy` command. 

Finally git clone pytorch-lightning and comment out scikit-learn, torch, torchvision in requirements.txt file. 
Note that pandas will take at least 30mins. 

Hope that helps.

**Comment by Borda** [comment]
Perfect, happy to hear it!
Btw Commenting out is not needed, since you have these packages they are skipped automatically...

**Comment by enod** [comment]
idk, for some reason without commenting out, it was still looking for torch, torchvision even though it's installed before.

**Comment by neggert** [comment]
FWIW, I think we have a loose plan to get rid of the pandas dependency.

**Comment by Borda** [comment]
https://github.com/PyTorchLightning/pytorch-lightning/issues/693

**Comment by ApluUalberta** [comment]
> I have managed to install and was able to run MNIST example.
> 
> Install `pytorch` and `torchvision` via https://devtalk.nvidia.com/default/topic/1049071/pytorch-for-jetson-nano-version-1-3-0-now-available/ which provides prebuilt binary pip wheels.
> 
> And install scipy with `sudo apt-get install python3-scipy` command.
> 
> Finally git clone pytorch-lightning and comment out scikit-learn, torch, torchvision in requirements.txt file.
> Note that pandas will take at least 30mins.
> 
> Hope that helps.

the requirements.txt has been changed and no longer has torchvision and scikit-learn as one of the requirements. However, it seems to seek a torch version>=1.4 as a result of torchmetrics>=0.2.0 (within requirements.txt). My Jetson even has torch 1.8.0 and torchvision in its pip3 package manager. Were you able to get it working with the new requirements?

My pip install attempt:
![Selection_001](https://user-images.githubusercontent.com/39037565/117083508-b812b500-acf9-11eb-9011-edff544c33b7.png)

my pip install requirements attempt:
![Selection_004](https://user-images.githubusercontent.com/39037565/117083528-c3fe7700-acf9-11eb-982e-9a88ea7bb614.png)

my pip list:
![Selection_005](https://user-images.githubusercontent.com/39037565/117083537-c82a9480-acf9-11eb-98f8-71c1ef9486fa.png)

Is there something I'm missing? I have also tried running setup.py to no avail. Thanks!
